setuptools/_distutils/command/sdist.py

```diff
@@ -268,7 +268,7 @@ def _add_defaults_standards(self):
                     self.warn("standard file '%s' not found" % fn)
 
     def _add_defaults_optional(self):
-        optional = ['test/test*.py', 'setup.cfg']
```

Comment:
Unfortunately, it's not allowed to make changes to distutils here. If you wish to make changes to distutils, they need to happen in the [pypa/distutils](/pypa/distutils) project.